### ISSUE TITLE
swift WCDB version v1.0.8.2 use .fts3 and .wcdb tokenize create english conent index "enumerateLinguisticTagsInRange" crashed。

### ISSUE THREAD

**Issue by BAOU** [Tencent/wcdb#1064]
### The language of WCDB

Swift

### The version of WCDB

v1.0.8.2

### The platform of WCDB

iOS

### The installation of WCDB

Cocoapods

### What's the issue?

> Post the outputs or screenshots for errors.

```
static let orthography = NSOrthography(dominantScript: "Latin", languageMap: [ "Latn": ["en"]])
    func lemmatization(input: UnsafePointer<Int8>, inputLength: Int32) -> Int32 {
        if inputLength > buffer.count {
            buffer.expand(toNewSize: Int(inputLength))
        }
        for i in 0..<Int(inputLength) {
            buffer[i] = UInt8(tolower(Int32(input.advanced(by: i).pointee)))
        }
        let optionalString = buffer.withUnsafeBytes { (bytes) -> String? in
            return String(bytes: bytes.baseAddress!.assumingMemoryBound(to: Int8.self),
                          count: Int(inputLength),
                          encoding: .ascii)
        }
        guard let string = optionalString else {
            return SQLITE_OK
        }
        var optionalLemma: String? = nil
        string.enumerateLinguisticTags(in: string.startIndex..<string.endIndex,
                                       scheme: NSLinguisticTagScheme.lemma.rawValue,
                                       options: NSLinguisticTagger.Options.omitWhitespace,
                                       orthography: WCDBCursorInfo.orthography,
                                       invoking: { (tag, _, _, stop) in
                                        optionalLemma = tag.lowercased()
                                        stop = true
        })
        guard let lemma = optionalLemma,
            lemma.count > 0,
            lemma.caseInsensitiveCompare(string) != ComparisonResult.orderedSame else {
            return SQLITE_OK
        }
        lemmaBufferLength = Int32(lemma.count)
        if lemmaBufferLength > lemmaBuffer.capacity {
            lemmaBuffer.expand(toNewSize: Int(lemmaBufferLength))
        }
        memcpy(&lemmaBuffer, lemma.cString, Int(lemmaBufferLength))
        return SQLITE_OK
    }
```

crash at `string.enumerateLinguisticTags(in: string.startIndex..<string.endIndex,`
`Thread 1: EXC_BREAKPOINT (code=1, subcode=0x1c272c6c0)`

> Explain what you want by example or code **in English**.

```
class SampleFTSData: TableCodable {
    var name: String?
    var content: String?
    enum CodingKeys: String, CodingTableKey {
        typealias Root = SampleFTSData
        static let objectRelationalMapping = TableBinding(CodingKeys.self)
        case name
        case content
        static var virtualTableBinding: VirtualTableBinding? {
            return VirtualTableBinding(with: .fts3, and: ModuleArgument(with: .WCDB))
        }
    }
}
```

```
do {
                //English
                let object = SampleFTSData()
                object.name = "content test"
                object.content = "content test test"
                try wcdb.insert(objects: object, intoTable: "SampleFTSData")
}
```
run to this code，then crashed：
`try wcdb.insert(objects: object, intoTable: "SampleFTSData")`

**Comment by Qiuwen-chen** [comment]
Fixed in the latest version.

**Comment by BAOU** [comment]
> Fixed in the latest version.

Thank you for your reply.
I don't want to upgrade to the latest version, which version fixes the issue?

**Comment by Qiuwen-chen** [comment]
It‘s fixed since v2.0.0

**Comment by BAOU** [comment]
> It‘s fixed since v2.0.0

Thanks!
